front/app/containers/IdeasShowPage/index.tsx

```diff
@@ -74,13 +74,6 @@ const IdeasShowPage = () => {
     phases?.data
   );
 
-  // If the idea is not in the current phase, we don't show the CTA bar
-  const ideaCurrentPhaseFilter = idea?.data.relationships.phases.data.filter(
-    (phase) => phase.id === participationContext?.id
-  );
-  const isIdeaInCurrentPhase =
-    ideaCurrentPhaseFilter && ideaCurrentPhaseFilter.length > 0;
-
```

Comment:
Moving these inside the if block reduces the need for checks (+ puts them closer to where they're used).